src/components/MessagesList/MessagesList.vue

```diff
@@ -627,6 +634,57 @@ export default {
 			}
 		},
 
+		handleMessageEdited(id) {
+			// regroup messagesGroupedByAuthor when a message is edited
```

Comment:
... and a one-liner here:
`this.messagesGroupedByAuthor = this.prepareMessagesGroups(this.messagesList)`